apps/nextjs-website/src/config.ts

```diff
@@ -41,6 +41,22 @@ export const defaultOgTagImage = `${baseUrl}/images/dev-portal-home.jpg`;
 export const resetResendEmailAfterMs = 4_000;
 export const fetchWebinarsQuestionsIntervalMs = 2_500;
 
+export const companyRoles = [
+  'ente-pubblico',
+  'partner-tecnologico',
+  'psp',
+  'gestore-di-pubblico-servizio',
+  'azienda-privata',
+  'altro',
+];
+
+export const signUpAdvantages = [
+  'exclusive_contents',
+  'product_updates',
+  'api_keys',
+  'support',
+];
+
```

Comment:
I would suggest keeping them inside the `_contents` folder as they contain information that will be determined by the editor

Reply:
oki